### DISCUSSION TITLE
Add Option to Disable SMS Delivery Report Requests

### DISCUSSION THREAD

**Discussion by EgisLTU** [capcom6/android-sms-gateway#152] [Ideas]
Many SMS services and mobile carriers charge extra for messages that include a delivery report request. This additional cost can be inconvenient for users who do not require delivery confirmations.

To address this, I propose adding an option to disable SMS delivery report requests in the application settings or API configuration. This feature would give users greater control over their SMS costs and usage preferences.

**Comment by capcom6** [comment]
Already implemented, please refer to the `withDeliveryReport` field of the request: https://capcom6.github.io/android-sms-gateway/#/Messages/post-message
